pkg/apis/metalkube/v1alpha1/baremetalhost_types.go

```diff
@@ -71,25 +71,27 @@ type BareMetalHostSpec struct {
 
 // CPU describes one processor on the host.
 type CPU struct {
-	Type  string `json:"type"`
-	Speed int    `json:"speed"` // GHz
+	Type     string `json:"type"`
+	SpeedGHz int    `json:"speedGHz"`
 }
 
 // Storage describes one storage device (disk, SSD, etc.) on the host.
 type Storage struct {
-	Size int    `json:"size"` // GB
-	Info string `json:"info"` // model, etc.
+	SizeGIB int    `json:"sizeGIB"`
```

Comment:
GIB here, by GB for NIC speed and RAM amount?  Intentional?

Reply:
Updated